finder-app/testfile.txt

```diff
@@ -1,3 +1 @@
-hello-world
-hello-world
-hello-world
+This-is-the-test-string-6
```

Comment:
@yogesh-1303 this file and the magic.mgc file above, writerarch below probably don't need to be in your repo.